src/main/java/com/conoftherings/util/DraftPackConstants.java

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
This may be replaced by a config file (or DB table). Did this for speed and simplicity at the moment.